src/components/Toolbar/GradientGenerator.svelte

```
@@ -1,18 +1,18 @@
 <script lang="ts">
-	import _ld from 'lodash';
-	import { ColorTranslator } from 'colortranslator';
-	import type { IconGradient } from '../../models/IconGradient';
-	import DeleteIcon from 'lucide-svelte/icons/trash';
-	import { customizedIcon } from '../../stores/icon-customizations.store';
-	import * as _iconPreview from '../../models/CustomizableIcon';
+	import _ld from "lodash";
```

ESLint

[error] '_ld' is defined but never used.

(@typescript-eslint/no-unused-vars)